### PR TITLE
[FIX] partner_company_group: module was splitted, this allows safe remove

### PR DESCRIPTION
**Pull request by yankinmax**
This PR is needed:
To avoid possible conflicts having both `partner_company_group` and `base_partner_company_group` with related in this repository.
I suppose the next step will be to replace `partner_company_group` by `base_partner_company_group` and related.

Depends on:
- [x] https://github.com/OCA/partner-contact/pull/1346
- [x] https://github.com/OCA/partner-contact/pull/1245
- [x] https://github.com/OCA/partner-contact/pull/1246
- [x] https://github.com/OCA/partner-contact/pull/1247

### CODE REVIEW

File: partner_company_group/views/contact_view.xml [hunk 1]
```diff
@@ -1,43 +1,24 @@
...
+        <field name="module">partner_company_group</field>
+        <field name="res_id" ref="base_partner_company_group.view_res_partner_filter" />
     </record>
-    <record id="action_open_group_members" model="ir.actions.act_window">
-        <field name="name">Company group members</field>
-        <field name="res_model">res.partner</field>
-        <field name="binding_model_id" ref="model_res_partner" />
-        <field name="view_mode">tree</field>
-        <field name="target">current</field>
+    <record id="action_open_group_members" model="ir.model.data">
```
Comment (gurneyalex) [inline partner_company_group/views/contact_view.xml]:
```suggestion
    <record id="xmlid_action_open_group_members" model="ir.model.data">
```

**Review by grindtildeath — APPROVED** [review]
Could be good to state in the readme that the module is an empty shell that will update XMLIDs.

Pre approving anyway.

**Comment by yankinmax** [comment]
@victoralmau @ernestotejeda

**Pull request merged**